tests/specs/axe-spec.js

```diff
@@ -0,0 +1,42 @@
+/* global browser, describe, it, expect, viewport */
+describe('axe', () => {
+  const viewports = viewport('tiny', 'huge');
```

Comment:
Can one of theses be updated to not use `viewports` specifically to hit this [case](https://github.com/cerner/terra-toolkit/pull/18/files#diff-f1b44da52f6b6b95ff1f06b8ca99cb3cR38).

Reply:
👍